NPE on parseName()

## Exception
* __User Action:__ UI Error
* __Request:__ Application crash
* __Version:__ 3.5.2
* __OS:__ Linux Android 9 - 28
* __Device:__ BV9500Plus
* __Model:__ BV9500Plus
* __Product:__ BV9500Plus_EEA
<details><summary><b>Crash log </b></summary><p>

```
java.lang.RuntimeException: An error occurred while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:354)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
at java.util.concurrent.FutureTask.run(FutureTask.java:271)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
at java.lang.Thread.run(Thread.java:764)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'long java.util.Date.getTime()' on a null object reference
at com.amaze.filemanager.filesystem.files.FileUtils.parseName(FileUtils.java:756)
at com.amaze.filemanager.filesystem.root.ListFilesCommand.parseStringForHybridFile(ListFilesCommand.kt:190)
at com.amaze.filemanager.filesystem.root.ListFilesCommand.listFiles(ListFilesCommand.kt:56)
at com.amaze.filemanager.asynchronous.asynctasks.LoadFilesListTask.doInBackground(LoadFilesListTask.java:222)
at com.amaze.filemanager.asynchronous.asynctasks.LoadFilesListTask.doInBackground(LoadFilesListTask.java:67)
at android.os.AsyncTask$2.call(AsyncTask.java:333)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
... 3 more

```
</details>
<hr>

--- a/app/src/main/java/com/amaze/filemanager/filesystem/files/FileUtils.java
+++ b/app/src/main/java/com/amaze/filemanager/filesystem/files/FileUtils.java
@@ -752,8 +752,12 @@ public class FileUtils {
       ParsePosition pos = new ParsePosition(0);
       SimpleDateFormat simpledateformat = new SimpleDateFormat("yyyy-MM-dd | HH:mm");
       Date stringDate = simpledateformat.parse(date, pos);
+      if (stringDate == null) {
+        Log.w(TAG, "parseName: unable to parse datetime string [" + date + "]");
+      }
       HybridFileParcelable baseFile =
-          new HybridFileParcelable(name.toString(), array[0], stringDate.getTime(), Size, true);
+          new HybridFileParcelable(
+              name.toString(), array[0], stringDate != null ? stringDate.getTime() : 0, Size, true);
       baseFile.setLink(link.toString());
       return baseFile;
     } else {

